### ISSUE TITLE
Compilation fails on recent CURRENT

### ISSUE THREAD

**Issue by arrowd** [hlh-restart/rtsx#12]
```
cc  -O2 -pipe -fno-strict-aliasing  -Werror -D_KERNEL -DKLD_MODULE -nostdinc   -include /wrkdirs/usr/ports/sysutils/rtsx-kmod/work/rtsx-1.0g/opt_global.h -I. -I/usr/src/sys -I/usr/src/sys/contrib/ck/include -fno-common  -fdebug-prefix-map=./machine=/usr/src/sys/i386/include -fdebug-prefix-map=./x86=/usr/src/sys/x86/include     -MD  -MF.depend.rtsx.o -MTrtsx.o -mno-mmx -mno-sse -msoft-float -ffreestanding -fwrapv -fstack-protector -Wall -Wredundant-decls -Wnested-externs -Wstrict-prototypes -Wmissing-prototypes -Wpointer-arith -Wcast-qual -Wundef -Wno-pointer-sign -D__printf__=__freebsd_kprintf__ -Wmissing-include-dirs -fdiagnostics-show-option -Wno-unknown-pragmas -Wno-error-tautological-compare -Wno-error-empty-body -Wno-error-parentheses-equality -Wno-error-unused-function -Wno-error-pointer-sign -Wno-error-shift-negative-value -Wno-address-of-packed-member -Wno-format-zero-length   -mno-aes -mno-avx  -std=iso9899:1999 -c rtsx.c -o rtsx.o
rtsx.c:634:12: error: format specifies type 'long' but the argument has type 'uint64_t' (aka 'unsigned long long') [-Werror,-Wformat]
                                              rtsx_read_count, rtsx_write_count);
                                              ^~~~~~~~~~~~~~~
rtsx.c:634:29: error: format specifies type 'long' but the argument has type 'uint64_t' (aka 'unsigned long long') [-Werror,-Wformat]
                                              rtsx_read_count, rtsx_write_count);
                                                               ^~~~~~~~~~~~~~~~
2 errors generated.
*** [rtsx.o] Error code 1
```

Full build log: http://beefy17.nyi.freebsd.org/data/head-i386-default/p545799_s364495/logs/rtsx-1.0.g,1.log

**Comment by hlh-restart** [comment]
I comment out the debug statement in c5f37c45e9e416c70695e9ba45a3ec4c8b728b84
I do it by phone... 
I will be back saturday.

**Comment by hlh-restart** [comment]
This issue should be solved by version 1.0h
